.circleci/config.yml

```diff
@@ -71,7 +71,7 @@ jobs:
 
     - run: |
         sed -i 's/version:.*/version: 1.0.0-'"${CIRCLE_SHA1}"'/' helm/cert-exporter-chart/Chart.yaml
-        ./architect publish --channels=stable
+        ./architect publish --pipeline=false --channels=stable
```

Comment:
`--pipeline=false` makes the publish command to actually publish to the channel `stable`, not `0-1-beta` like before.